components/board/genericlinux/i2c.go

```diff
@@ -0,0 +1,125 @@
+package genericlinux
+
+import (
+	"context"
+	"encoding/binary"
+	"sync"
+
+	"github.com/edaniels/golog"
+	"periph.io/x/conn/v3/i2c"
+	"periph.io/x/conn/v3/i2c/i2creg"
+	"periph.io/x/host/v3"
+
+	"go.viam.com/rdk/components/board"
+)
+
+func init() {
+	if _, err := host.Init(); err != nil {
```

Comment:
Part of me says we should call `host.Init()` here because this is the file that uses it, and part of me says we can skip it because every board that uses genericlinux calls it themselves (e.g., jetson/board.go, beaglebone/board.go, ti/board.go). If it's here, I wonder if we should remove it from the other places. What do you think?

Reply:
This initializes periph.io's registries of things you can get later. This is the file that gets things later, not jetson/board.go and friends, so this feels like the right file to initialize the registry to me. but I'm open to disagreement: why would you prefer it to be in the other places?

Reply:
Just saw your edit, that was my main concern. We shall confirm with tests! (Software or real)